packages/manager/apps/pci-private-registry/src/pages/CIDR/useDatagridColumn.tsx

```diff
@@ -0,0 +1,170 @@
+import {
+  DatagridColumn,
+  DataGridTextCell,
+} from '@ovh-ux/manager-react-components';
+import { useCallback, useMemo } from 'react';
+
+import { useParams } from 'react-router-dom';
+import { useQueryClient } from '@tanstack/react-query';
+
+import { useTranslation } from 'react-i18next';
+import { FormProvider, useFormContext } from 'react-hook-form';
+import ButtonsCIDR from '@/components/CIDR/ButtonsCIDR.component';
+import { TIPRestrictionsData } from '@/types';
+import {
+  getRegistryQueyPrefixWithId,
+  useIpRestrictionsWithFilter,
+} from '@/api/hooks/useIpRestrictions';
+import ActionComponent from '@/components/CIDR/Actions.component';
+import Authorization from '@/components/CIDR/Authorization.component';
+import Description from '@/components/CIDR/Description.component';
+import IpBlock from '@/components/CIDR/IpBlock.component';
+import { capitalizeAndJoin } from '@/helpers';
+import AuthorizationLabel from '@/components/CIDR/AuthorizationLabel.component';
+import Checkboxes from '@/components/CIDR/Checkboxes.component';
+import AllCheckboxComponent from '@/components/CIDR/AllCheckbox.component';
+import useFilter from './useFilters';
+
+function evaluateDraftAndData(draft: boolean, dataLength: number) {
```

Comment:
I don't understand what this function is used for, could you rename it ?

Reply:
to show checkbox or not depending on minimum 3 element including a draft one or two if no draft